app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -346,24 +338,6 @@ class ProfileEditActivityTest {
     }
   }
 
-  @Test
-  fun testProfileEdit_updateName_checkNewNameDisplayed() {
```

Comment:
I'm a bit confused by this change--isn't the PR only changing the activity's accessibility label not its title?

Reply:
It is changing both the title and the label. Sean suggested we change the behaviour of the activity so that the title shown is **_Edit Profile_** and not the name of the profile being displayed.

Reply:
Here is a brief snapshot of the conversation I had with Sean about the issue.

> Kenneth Murerwa, Thu 5:24 PM
> Hey @Sean Lip and @Ben Henning. I am having a bit of an issue here. I am currently working on issue #3602 (https://github.com/oppia/oppia-android/issues/3602) and while writing a test for the issue, I realized a conflict in the activity behavior.  Currently, the ProfileEditActivity has the name of the profile being edited on the action bar instead of the label/title (screenshot attached). If I write a test to check that the label is displayed, it always fails since the title is overwritten by the profile name. Is this the accepted behavior (and therefore I should not make any changes to the test since a test for that already exists) or should I ensure that the label (currently Edit Profile) should be shown instead of the profile name?
> 
> ![Screenshot of the current screen](https://user-images.githubusercontent.com/18438114/224654121-2fd9d818-1ac6-4190-bf34-ae2ca2652959.png)
> 
> Sean Lip, Thu 5:31 PM
> Let's show Edit Profile in the navbar, since the profile name is already shown just below it. Thanks for checking!

Reply:
I think it's fine that we make the title & label the same, but we should then be following the approach similar to here: https://github.com/oppia/oppia-android/blob/087f2fe85aa01655ad03bbebbc64ab5edd13089f/app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt#L173 where:
- The string name is "title" not "label"
- The test verifies the toolbar title

Reply:
This looks to be resolved in the latest changes.